lang/python/python-mysql/Makefile

```diff
@@ -1,20 +1,20 @@
 #
-# Copyright (C) 2007-2014 OpenWrt.org
+# Copyright (C) 2007-2018 OpenWrt.org
 #
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
 
 include $(TOPDIR)/rules.mk
 
-PKG_NAME:=MySQL-python
-PKG_VERSION:=1.2.5
+PKG_NAME:=mysqlclient
+PKG_VERSION:=1.3.12
 PKG_RELEASE:=1
 PKG_LICENSE:=GPL-2.0
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).zip
```

Comment:
could you use `tar.gz` instead of `.zip` ? :)
both would be fine, but there is a preference to using `tar.gz` if available